runners/spark/src/main/java/org/apache/beam/runners/spark/SparkRunner.java

```diff
@@ -142,6 +143,8 @@ private void registerMetrics(final SparkPipelineOptions opts, final JavaSparkCon
             : Optional.<CheckpointDir>absent();
     final Accumulator<NamedAggregators> aggregatorsAccumulator =
         SparkAggregators.getOrCreateNamedAggregators(jsc, maybeCheckpointDir);
+    // Instantiate metrics accumulator
+    MetricsAccumulator.getOrCreateInstance(jsc, maybeCheckpointDir);
```

Comment:
So I guess this is an init() ? no use for the return value, and other places use getInstance() and even throw if not instantiated. Am I missing something ?